### PR TITLE
fix cmake changes for script copy for dev builds

### PR DESCRIPTION
**Pull request by birarda**

### CODE REVIEW

File: interface/CMakeLists.txt [hunk 1]
```diff
@@ -326,27 +326,28 @@ if (APPLE)
...
     # the ttf files, but doing so generates a CORS policy violation,
     # so we have to retain a copy of the fonts outside of the resources binary
     COMMAND "${CMAKE_COMMAND}" -E copy_directory
       "${PROJECT_SOURCE_DIR}/resources/fonts"
-      "$<TARGET_FILE_DIR:${TARGET_NAME}>/resources/fonts"
+      "${RESOURCES_DEV_DIR}/fonts"
     COMMAND "${CMAKE_COMMAND}" -E copy_directory
       "${CMAKE_SOURCE_DIR}/scripts"
-      "${RESOURCES_DEV_DIR}/scripts"
+      "${INTERFACE_EXEC_DIR}/scripts"
```
Comment (birarda) [inline interface/CMakeLists.txt]: This is what ended up being changed - we were copying scripts to resources and not beside the exe

**Comment by birarda** [comment]
We'll need this for RC66 too (even though it should affect local dev builds only)

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8334/app_12732_PR.apk

**Comment by AndrewMeadows** [comment]
I tested this PR on linux: it fixes the problem I was experiencing after pulling recently from `hifi/master`: "unable to load defaultScripts.js".

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8334/HighFidelity-Beta-PR12732-0172c24cf2f3a2557c56e0cef46cfc6157dc12fb.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8334/HighFidelity-Beta-PR12732-0172c24cf2f3a2557c56e0cef46cfc6157dc12fb.dmg

**Comment by birarda** [comment]
Since this PR fixes an issue seen only in local builds I am using my personal clean build test and Andrew's as a QA approval

**Pull request merged**